dispersy.py

```diff
@@ -693,6 +715,8 @@ def get_communities(self):
         """
         return self._communities.values()
 
+    @inlineCallbacks
+    # TODO(Laurens): This method is never called?
```

Comment:
It could be called dynamically, like we've seen some times before in Dispersy. Can you figure this out?